File: app/src/main/java/com/example/ctap/fido2/PublicKeyCredentialUserEntity.java
package com.example.ctap.fido2;

import com.fasterxml.jackson.annotation.JsonProperty;


public class PublicKeyCredentialUserEntity {
    @JsonProperty("id") public byte[] id;
    @JsonProperty("name") public String name;
    @JsonProperty("displayName") public String displayName;
}

File: app/src/main/java/com/example/ctap/fido2/PublicKeyCredentialUserEntity.kt
package com.example.ctap.fido2

import com.fasterxml.jackson.annotation.JsonAutoDetect
import com.fasterxml.jackson.annotation.JsonProperty

@JsonAutoDetect(fieldVisibility = JsonAutoDetect.Visibility.ANY)
internal class PublicKeyCredentialUserEntity (
        @JsonProperty("id") var id: ByteArray,
        @JsonProperty("name") var name: String,
        @JsonProperty("displayName") var displayName: String
)